test/build_test/utils_test.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I wonder if this should live under build_test/test? We would then need to add something to the travis file as well to run the tests under there.